gef.py

```diff
@@ -2995,7 +3008,7 @@ def mprotect_asm(cls, addr: int, size: int, perm) -> str:
 
 @register_architecture
 class MIPS(Architecture):
-    aliases = ("MIPS", Elf.MIPS)
+    aliases: Tuple[Union[str, int], ...] = ("MIPS", Elf.MIPS)
```

Comment:
same Q about putting the redundant hint here. If we actually need it, then it's missing for riscv

Reply:
same Q, so same A :P